R/shiny_prerendered.R

```diff
@@ -69,11 +69,12 @@ shiny_prerendered_app <- function(input_rmd, render_args) {
   }
 
   # attach dependencies to final html
-  html <- htmltools::attachDependencies(html, deps)
+  html_doc <- htmltools::htmlTemplate(text_ = html, document_ = TRUE)
```

Comment:
I think we don't actually want template expansion to happen here, right? Otherwise if your Rmd happened to contain `{{blah}}` we'd try to evaluate it as R code! We just want the effects of `document_ = TRUE`. It might be better to just do that directly:

```r
html_doc <- tagList(HTML(html))
class(html_doc) <- c("html_document", class(html_doc))
```